common/extension/proxy_factory.go

```diff
@@ -1,6 +1,22 @@
+// Copyright 2016-2019 hxmhlt & Yincheng Fang
```

Comment:
Please use standard Apache license.

Reply:
Hi, I think fixing the dead link it more important, please fix it in the next pull request.